### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Mango-3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## Overview
 
-appleseed-maya is an appleseed plugin for [Autodesk® Maya®](http://www.autodesk.com/products/maya/overview) 2017 and 2018.
+appleseed-maya is an appleseed plugin for [Autodesk® Maya®](http://www.autodesk.com/products/maya/overview) 2018, 2019 and 2020.
```
Comment (dictoon) [inline README.md]: We probably should clarify that 2018 is not supported on Linux:

    ... for Maya 2018 (Windows only), 2019 and 2020.

**Review by dictoon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Mango-3) [reply README.md]: Done

**Comment by Mango-3** [comment]
There is also a PR for the appleseedhq website to update the plugin versions.

**Review by dictoon — APPROVED** [review]
Perfect, thanks!

**Pull request merged**